### PR TITLE
fixup! Update operators and dtypes to follow numpy 1.10 changes

### PR DESCRIPTION
**Pull request by laem**

### CODE REVIEW

File: openfisca_france/model/prelevements_obligatoires/prelevements_sociaux/cotisations_sociales/apprentissage.py [hunk 1]
```diff
@@ -85,9 +85,10 @@ def function(self, simulation, period):
         for age_interval in salaire_en_smic:
             age_condition = (age_interval["age_min"] <= age) * (age < age_interval["age_max"])
             output[age_condition] = sum([
-                (anciennete_contrat[age_condition] == anciennete) * part_de_smic
+                (anciennete_contrat[age_condition] == np.array([anciennete], dtype = 'timedelta64[Y]')) * part_de_smic
                 for anciennete, part_de_smic in age_interval['part_de_smic_by_anciennete'].iteritems()
                 ])
+        print output
```
Comment (cbenz) [inline openfisca_france/model/prelevements_obligatoires/prelevements_sociaux/cotisations_sociales/apprentissage.py]: There is still a print :)

Comment (cbenz) [inline openfisca_france/model/prelevements_obligatoires/prelevements_sociaux/cotisations_sociales/apprentissage.py]: @laem Don't update the PR for that, I'll handle it.

**Comment by cbenz** [comment]
Merged via 1f02d67439a6c41992930e9b984bf13beb77e41b

File: openfisca_france/model/prelevements_obligatoires/prelevements_sociaux/cotisations_sociales/apprentissage.py (hunk 1 above)
Comment (laem) [inline openfisca_france/model/prelevements_obligatoires/prelevements_sociaux/cotisations_sociales/apprentissage.py]: Looks like it's still there :disappointed_relieved: @cbenz

**Pull request closed without merging**